sqlite.lisp

```diff
@@ -0,0 +1,122 @@
+(in-package #:cl)
+
+(defpackage #:scheduler-sqlite
+  (:use #:cl #:alexandria #:sqlite #:iterate)
+  (:import-from #:scheduler
+                #:sqlite-scheduler)
+  (:import-from #:scheduler-implementation
+                #:scheduler #:in-memory-scheduler 
+                #:start-scheduler #:stop-scheduler
+                #:create-scheduler-task
+                #:read-scheduler-task
+                #:update-scheduler-task
+                #:delete-scheduler-task
+                #:list-scheduler-tasks
+                ;; entry
+                #:scheduler-entry
+                #:cron-entry
+                #:parse-entry
+                #:parse-cron-entry
+                ;; task
+                #:task
+                #:task-time-specs
+                #:task-command
+                #:task-last-execution
+                #:task-next-execution
+                #:task-source-entry
+                #:execute-task)
+  (:export #:sqlite-scheduler
+           ;; task
+           #:sqlite-task))
+
+(in-package #:scheduler-sqlite)
+
+(defvar *db-file-path*
+  "Path to the SQLite DB file.")
```

Comment:
please remove the global variable and store it as a slot in the `sqlite-scheduler` instance, otherwise you limit yourself to only one database at the same time.

Reply:
Done